cmd/start/main.go

```diff
@@ -200,7 +200,7 @@ func main() {
 		}
 	}
 
-	if len(ctxt.Environment) > 0 {
+	if ctxt.Environment != "" {
```

Comment:
No, both will always work. Go never has "undefined" values, but every type has a "zero value". `ctxt.Environment` is a `string`, and the zero value of a string is `""`. It is idiomatic Go to compare with the zero value if you want to check if sth. is set. Hence, I changed to string comparison instead of length, even though both works exactly the same!